content/raw/chat/username_avatar.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "username & profile spec"
+description: ""
+date: 2020-12-06T08:48:57+00:00
+lastmod: 2020-12-06T08:48:57+00:00
+draft: false
+images: []
+menu:
+  raw:
+    parent: "chat"
+weight: 100
+toc: true
+---
+
+# Username & Profile Spec
+
+## Motivation
+
+Users need a mechanism to identity themselves to other users that is easy to use and at the same time somewhat resistant to impersonation attacks. At the time of writing Status uses deterministic 3 word usernames and ENS usernames. Unfortunatly 3 word usernames have shown to be quite confusing for new users as they are used to set their own usernames in any chat app.
+Here we introduce support for users to set their own username and introduce at least 3 mechaniscs to minimize the risk of impersonation attacks. The first mechanism is the use of deterministic sequence of emojis to uniquely identity a profile, the second mechanism is the use of a deterministic sequence of colors around the profile to differentiate it from other profiles which effectively replaces the 3 word usernames and identicons with a similar deterministic identifier, the third mechanism involves preventing duplicate usernames from joining the same community. There is also an identity verification mechanism that is used to ensure that the user is who they say they are described in the Identity Verifcation Spec.
+
+## Designs
+
+- [designs](https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=1837%3A335126)
+
+## Use Cases
+
+## Functional Requirements
+
+**Profile Emblems:**
+
+- If a profile is a mutual contact then it MUST display a mutual contact emblem (e.g a profile icon) and it MUST NOT display other emblems.
```

Comment:
This is slightly incorrect, a mutual contact can also display the untrustworthy emblem.  This is the only scenario where a contact can display two emblems